components/publish/PublishWidget.vue

```diff
@@ -63,14 +63,25 @@ const { editor } = useTiptap({
   onPaste: handlePaste,
 })
 
+function trimPollOptions() {
+  const indexLastNonEmpty = draft.params.poll!.options.findLastIndex(option => option.trim().length > 0)
+  const trimmedOptions = draft.params.poll!.options.slice(0, indexLastNonEmpty + 1)
+
+  if (currentInstance.value?.configuration
+      && trimmedOptions.length >= currentInstance.value?.configuration?.polls.maxOptions)
+    draft.params.poll!.options = trimmedOptions
+  else
+    draft.params.poll!.options = [...trimmedOptions, '']
+}
```

Comment:
I think this is ok for now, but maybe later we could leave the extra options with the `5/4` in red so the user can decide what options to trim? It should be rare to have this issue though.